lib/discordrb/data/user.rb

```diff
@@ -166,7 +166,18 @@ def webhook?
       @discriminator == Message::ZERO_DISCRIM
     end
 
-    %i[offline idle online].each do |e|
+    # @!group Statuses
+    # @!method offline?
+    #   @return [true, false] whether this user is offline.
+    # @!method idle?
+    #   @return [true, false] whether this user is idle.
+    # @!method online?
+    #   @return [true, false] whether this user is online.
+    # @!method dnd?
+    #   @return [true, false] whether this user is on Do Not Disturb.
```

Comment:
not sure how I feel about the pascal casing for `do not disturb`. maybe something like this?
```suggestion
    #   @return [true, false] whether this user is set to do not disturb.
```
I'm sure this is opinionated though